bot/exts/utils/extensions.py

```diff
@@ -0,0 +1,196 @@
+            check_extension = ext.split(".")
+            if argument == check_extension[len(check_extension) - 1]:
+                matches.append("".join(ext))
+        
+        return " ".join(matches)
+
+
+class Extensions(commands.Cog):
+    """Commands for extension management."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @commands.group(
+        name="extensions",
+        aliases=["ext", "exts", "cog"],
+        invoke_without_command=True,
+    )
+    async def extension_group(self, ctx: commands.Context) -> None:
+        """Load, unload, reload, and list loaded extensions."""
+        await ctx.send_help(ctx.command)
+
+    @extension_group.command(name="load", aliases=["l"])
+    async def loading(self, ctx: commands.Context, *extensions: Extension) -> None:
+        """Loads the given extensions.
+        If `*` is provided as the extension then loads every single extension.
+        To reload the extension of an entire folder, `**` must be before the folder name.
+        """
+        if not extensions:
+            await ctx.send_help(ctx.command)
+            return
+                        
+        if '*' in extensions:
+            extensions = e.EXTENSIONS
+
+        messages = self.manage("load", extensions)
+        await ctx.channel.send(messages)
+
+    @extension_group.command(name="reload", aliases=["rl"])
+    async def reloading(self, ctx: commands.Cog, *extensions: Extension) -> None:
+        """
+        Reloads the given extensions.
+        If `*` is provided as the extension then reloads all the extensions.
+        To reload the extension of an entire folder, `**` must be before the folder name.
+        """
+        if not extensions:
+            await ctx.send_help(ctx.command)
+            return
+        
+        if '*' in extensions:
+            extensions = e.EXTENSIONS
+
+        messages = self.manage("reload", extensions)
+        await ctx.channel.send(messages)
+
+    @extension_group.command(name="unload", aliases=["ul"])
+    async def unloading(self, ctx: commands.Cog, *extensions: Extension) -> None:
+        """
+        Unloads the given extensions.
+        If `*` is provided as the extension then unloads every all the extensions.
+        To reload the extension of an entire folder, `**` must be before the folder name.
+        """
+        if not extensions:
+            await ctx.send_help(ctx.command)
+            return
+        
+        if '*' in extensions:
+            extensions = [  ext for ext in e.EXTENSIONS 
+                            if ext not in UNLOAD_BLACKLIST
+                        ]
+
+        blacklists = [
+            blacklisted_extesnion
+            for blacklisted_extesnion in extensions
+            if blacklisted_extesnion in UNLOAD_BLACKLIST
+        ]
+        
+        if blacklists:
+            await ctx.channel.send(
+                f":x: `{' ,'.join(blacklists)}` extensions may not be unloaded"
```

Comment:
```suggestion
                f":x: `{' ,'.join(blacklists)}` extensions can not be unloaded"
```
"may" suggests an uncertainty on whether the extension is loaded.